apps/user-office-frontend/src/components/proposal/ProposalParticipant.tsx

```diff
@@ -73,12 +68,15 @@ export default function ProposalParticipant(props: {
         />
 
         <Tooltip title="Edit Principal Investigator">
-          <IconButton
-            onClick={() => setIsPickerOpen(true)}
-            className={classes.addButton}
-          >
-            <EditIcon data-cy="edit-proposer-button" fontSize="small" />
-          </IconButton>
+          <span>
```

Comment:
I think this `<span>` element is not needed here

Reply:
It has to be left in unfortunately like Fredrik was saying earlier:

> Tooltip.js:280 MUI: You are providing a disabled `button` child to the Tooltip component.
A disabled element does not fire events.
Tooltip needs to listen to the child element's events to display the title.
>
> Add a simple wrapper element, such as a `span`.